lms/static/styles/components/_SubmitGradeForm.scss

```diff
@@ -1,6 +1,16 @@
 @use "../mixins";
 @use "../variables" as var;
 
+@keyframes fadeAway {
```

Comment:
We don't have an established naming convention for animation keyframes. Since animation names are global, I suggest that we include the component name in any keyframes which are defined as part of a component, as opposed to being utility/shared animation names. One option would be to name it just like a CSS class (`SubmitGradeForm__some-name`).